<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="plot_page_scroll">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>635</width>
        <height>789</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="plot_type_selection_layout">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Plot type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="plot_type_selection">
           <item>
            <property name="text">
             <string>Confusion matrix</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Calibration curve</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DET curve</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Precision-recall curve</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ROC curve</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="plot_parameters_box">
         <property name="title">
          <string>Plot parameters</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Deposits</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="predictions_label">
            <property name="text">
             <string>Predictions (classifications)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Output file</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsMapLayerComboBox" name="deposits"/>
          </item>
          <item row="1" column="1">
           <widget class="QgsMapLayerComboBox" name="predictions"/>
          </item>
          <item row="5" column="1">
           <widget class="QgsFileWidget" name="output_file">
            <property name="storageMode">
             <enum>QgsFileWidget::SaveFile</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Save DPI</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="number_of_bins_label">
            <property name="text">
             <string>Number of bins</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QgsSpinBox" name="number_of_bins">
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QgsSpinBox" name="save_dpi">
            <property name="specialValueText">
             <string>Default</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="clearValue">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="probabilities_label">
            <property name="text">
             <string>Predictions (probabilities)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QgsMapLayerComboBox" name="probabilities"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="button_box">
         <property name="standardButtons">
          <set>QDialogButtonBox::NoButton</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="plot_container">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="plot_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
